study-builder/studies/pancan/emails/consent-complete.en.html

```diff
@@ -0,0 +1,15 @@
+#set( $text = {
+    "subject": "Thank you for providing your consent",
+    "salutation_self": "Dear Participant",
+    "salutation_parent": "Dear Parent",
+    "thank_you": "Thank you for joining Count Me In and for giving us your consent for research.",
+    "what_to_expect": "We will next mail you a kit with instructions on how to provide a saliva sample.",
+    "what_to_expect_sample_request": "In some cases, we may also contact you about providing a blood sample (1 tube or 2 teaspoons per sample), which can be drawn at your physician's office, local clinic, or nearby lab facility. If applicable, we will provide detailed instructions on how to do this. We may obtain copies of {{~#if proxy_firstName~}}your child's{{~else~}}your{{~/if~}} medical records and, after reviewing them, may also request a small portion of stored tumor sample from the hospital where it is kept. Once we have analyzed {{~#if proxy_firstName~}}your child's{{~else~}}your{{~/if~}} requested samples, we will return any unused tumor samples to the pathology department that sent them to us.",
+    "next_steps": "If you have not yet provided information about where {{~#if proxy_firstName~}}your child has{{~else~}}you have{{~/if~}} received care for your cancer(s), you can complete the process here:",
+    "button_title": "Medical Release",
+    "contact": "For your convenience, we have attached a copy of your signed consent form. If you have questions at any point and would like to talk to a member of our study staff, please reach out to us at <a href='mailto:${contact.email}' target='_blank' style='color: #7154ff; text-decoration: none; cursor: pointer;'>info@joincountmein.org</a> or call <a href='tel:${contact.phone}' style='color: #7154ff; text-decoration: none; cursor: pointer;>000-000-0000</a>.",
+    "closing": "Sincerely,",
+    "study_team": "The Count Me In team",
+    "footnote": "For additional updates, follow us on <a href='${social.twitter}' target='_blank' style='color: #7154ff; text-decoration: none; cursor: pointer;'>Twitter</a>, <a href='${social.facebook}' target='_blank' style='color: #7154ff; text-decoration: none; cursor: pointer;'>Facebook</a>, and <a href='${social.instagram}' target='_blank' style='color: #7154ff; text-decoration: none; cursor: pointer;'>Instagram</a>."
```

Comment:
The email text refers to it, but CMI is asking us to leave out Instagram links out. They don't have anything for it yet and they are not sure when this might happen.

Reply:
Fixed in 93515815cbd59ad2ef82df671a5da84459ae0b9b